src/sentry/static/sentry/app/views/planout/installReact.jsx

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+
+import {t} from 'app/locale';
+
+const InstallReactTest = createReactClass({
+  displayName: 'ProjectInstallPlatform',
+
+  propTypes: {
+    dsn: PropTypes.str,
+  },
+
+  getInitialState(props) {
+    return {
+      loading: true,
+      error: false,
+    };
+  },
+
+  render() {
+    let dsn = this.props.dsn;
+
+    return (
+      <div>
+        <h3> Installation </h3>
+        <p>
+          {' '}
+          Start by adding the <span className="pre"> raven.js </span> script tag to your
+          page. It should be loaded as early as possible, before your main javascript
+          bundle.{' '}
+        </p>
+        <pre>
+          <span>{'<'}</span>
+          <span className="script">script </span>
+          <span className="attribute">src</span>
+          <span>=</span>
+          <span className="value-text">
+            "https://cdn.ravenjs.com/3.24.0/raven.min.js"
```

Comment:
i think we have a way to insert the latest version here dynamically (it might be the same thing that powers the "you're not on the latest version" helper texts)